lib/platform/bitbucket-server/index.js

```diff
@@ -122,6 +123,12 @@ async function initRepo({
   const [projectKey, repositorySlug] = repository.split('/');
   config = { projectKey, repositorySlug, gitPrivateKey };
 
+  /* istanbul ignore else */
+  if (bbUseDefaultReviewers !== false && opts.bbUseDefaultReviewers !== false) {
```

Comment:
I can't think of a scenario where you'd need to do this via hostRules. Renovate works on only one host at a time anyway, so by configuring it in regular config you are implicitly enabling it for the platform host.

Reply:
Ok, removing the `opts` part.